wdl/transforms/biscayne/src/main/scala/wdl/transforms/biscayne/linking/expression/types/BiscayneTypeEvaluators.scala

```diff
@@ -121,6 +121,32 @@ object BiscayneTypeEvaluators {
       ) mapN { (_, _) => WomArrayType(WomStringType) }
   }
 
+  implicit val quoteFunctionEvaluator: TypeEvaluator[Quote] = new TypeEvaluator[Quote] {
+    override def evaluateType(a: Quote, linkedValues: Map[UnlinkedConsumedValueHook, GeneratedValueHandle])(implicit
+      expressionTypeEvaluator: TypeEvaluator[ExpressionElement]
+    ): ErrorOr[WomType] =
+      validateParamType(a.param, linkedValues, WomArrayType(WomAnyType)) flatMap {
+        case WomArrayType(_: WomPrimitiveType) => WomArrayType(WomStringType).validNel
```

Comment:
If I recall, an empty array becomes `WomArrayType(WomNothingType)`, which doesn't seem to match any of the valid cases, despite it being valid input. Should that be handled here?